the-experts-fe/src/App.js

```diff
@@ -14,7 +14,7 @@ export default function App() {
   const [selectedContractors, setSelectedContractors] = useState([]);
 
   useEffect(() => {
-    fetch(`http://localhost:9000/contractor-results/${jobValue}`)
+    fetch(`/contractor-results/${jobValue}`)
       .then(res => res.text())
```

Comment:
Remember you should use `res.json()` instead of `res.text()` here, because the content-type of the returned data is JSON.